src/reportAppAzureDirective.js

```diff
@@ -0,0 +1,29 @@
+//this is generated for azure
+jgivenReportApp.directive('a', function () {
+    return {
+        restrict: 'E',
+        link: function (scope, elem, attrs) {
+            if (!attrs.href || attrs.target) {
+                return;
+            }
+
+            if (!attrs.href.includes('jgiven.org')) {
+                elem.on('click', function (e) {
+                    e.preventDefault();
+                    var newHref = '';
+                    if (attrs.href.includes(`{{`)) {
+                        newHref = attrs.href.slice(2, attrs.href.length - 2);
+                    } else {
+                        newHref = attrs.href;
+                    }
+                    scope.redirectTo(newHref);
+                });
+            } else {
+                elem.on('click', function (e) {
+                    e.preventDefault();
+                    window.open('https://jgiven.org', '_blank');
+                });
+            }
+        }
+    }
+})
```

Comment:
The idea is nice :) 
Although I guess there is quite some more code inside our app.bundle.js. 
As far as I can see we need all the stuff from line 98524 to line 98611.

I am not sure whether all lines in between are needed or whether there is any duplicate, although I will check this!

Reply:
Damn, I just saw that this code is not on top-level, but inside a function! That makes everything worse.

Reply:
We know which function is manipulated, so maybe it is possible to insert the necessary code via some fancy usage of sed. However, a 'nice' solution is something different indeed.

Reply:
I might havefound a solution. But it is ten times more hacky :(

Reply:
Before merging I am going to test the extension in DevOps :)

Reply:
After removing a few unnecessary dependencies the plugin builds and a test pipeline succeeded.

However, Azure DevOps won't display the results anymore (stuck in an endless loading procedure). 
I will investigate whether this is due to some lazyness on AzureDevops-Side or whether we destroyed some important functionality with this PR.

Therefore, for now we should not merge.

Reply:
you mean node_modules/jgiven-html-app/dist? yes, that should be filled by the yarn build step! if that does not happen the app won't be copied to the toplevel.

Reply:
Exactly, this folder seems to stay empty. The build seems to be triggered although this folder stays empty for some reason.

Reply:
Funnily enough, using an older version like 0.16.2 includes the needed files.

Reply:
The yarn build command will generate a new app.bundle.js which is actually a lot smaller than the one delivered via npm install.
I suspect there are few missing commands in this new app.bundle.js which lead to the malfunctioning Azure plugin. Therefore we should maybe think about some sort of PostBuild script to manipulate the app.bundle.js directly, although this also seems to be rather annoying.

Reply:
weird...